### PR TITLE
  RK-12478 Added a devcontainer. Updated the SDK

### PR DESCRIPTION
**Pull request by Urook**
Added a devcontainer.
Updated the sdk in the dockerfile and removed the -d flag.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ ADD . .
 
 # Because of the GoRook deployment architecture, the go.mod is pointing to the stub version and the CI is responsible for replacing it with the real version
 # We download the GoRook explicitly so it would register it in the go.sum
-RUN go get -d github.com/Rookout/GoSDK@v0.1.8
+RUN go get github.com/Rookout/GoSDK@v0.1.9
```
Comment (eli-hatamov-dt) [inline Dockerfile:15]: Why did you remove the `-d` flag?

Comment (Urook) [reply Dockerfile:15]: > Why did you remove the `-d` flag?

It is deprecated and used by default in go 1.18

**Review by EliRookout — APPROVED** [review]
